src/bosh-virtualbox-cpi/bin/test

```diff
@@ -5,14 +5,18 @@ result=0
 bin=$(dirname $0)
 
 echo -e "\n Formatting packages..."
-go fmt github.com/cppforlife/bosh-virtualbox-cpi/...
+go fmt bosh-virtualbox-cpi/...
 let "result+=$?"
```

Comment:
Maybe it makes sense to use `set -e` instead of the `result+=$?` pattern. It reduces conditionals, and ensures the script fails early.

Example: https://github.com/cloudfoundry/bosh-agent/blob/main/bin/ginkgo#L2